.ci/magician/cmd/scheduled_pr_reminders_test.go

```diff
@@ -0,0 +1,897 @@
+package cmd
+
+import (
+	"testing"
+	"time"
+
+	membership "magician/github"
+
+	"github.com/google/go-github/v61/github"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNotificationState(t *testing.T) {
```

Comment:
~are we running these unit tests anywhere?~

nvm, looks like the tests are included in membership-checker unit tests. The GHA is running all magician unit tests.